.pre-commit-hooks.yaml

```
@@ -82,7 +82,7 @@
   description: Validates all Terragrunt configuration files.
   entry: terragrunt_validate.sh
   language: script
-  files: (\.hcl)$
+  files: (/terragrunt\.hcl)$
```

Irrespective to the goal of the PR: parentheses are redundant here (as there's no grouping), forward slash needs to be escaped (as it is a special char in regex), and possibly files w/o path need to be taken into account (?):
```suggestion
  files: (^|\/)terragrunt\.hcl$
```